docker/Dockerfile

```diff
@@ -2,6 +2,13 @@
 
 FROM mcr.microsoft.com/vscode/devcontainers/rust:0-1
 
+# Install Starship https://starship.rs/ for pretty prompt
+RUN curl -fsSL https://starship.rs/install.sh | bash /dev/stdin --yes \
+    && echo 'eval "$(starship init zsh)"' >> /root/.zshrc
+
 ENV WASMTIME_VERSION=0.29.0
 RUN curl -L https://github.com/bytecodealliance/wasmtime/releases/download/v${WASMTIME_VERSION}/wasmtime-v${WASMTIME_VERSION}-x86_64-linux.tar.xz \
   | tar -xJ --wildcards --no-anchored --strip-components 1 -C /usr/bin wasmtime
+
+COPY entrypoint.sh /entrypoint.sh
+ENTRYPOINT ["/entrypoint.sh"]
```

Comment:
It's weird to me to put CI/CD in a script that's setup as the entrypoint rather than putting the commands in the ci/cd pipeline (i.e. in action.yaml or main.yaml).  It's fine to write a script for the commands to run but just weird to have it registered as an entrypoint.  I don't see how this script aligns with all uses of the docker container.

It also seems super weird since all cases of running the Dockerfile already includes access to the source repo and thus the script... so just run the script?

I must be missing something